public/js/common.js

```diff
@@ -260,6 +277,14 @@ LRR.buildThumbnailDiv = function (data, tagTooltip = true) {
     const id = data.arcid || data.id;
     let reader_url = new LRR.apiURL(`/reader?id=${id}`);
 
+    let bookmarkIcon = ``;
+    if ( LRR.bookmarkLinkConfigured() ) {
+        const isBookmarked = JSON.parse(localStorage.getItem("bookmarkedArchives") || "[]").includes(id);
+        const bookmarkClass = isBookmarked ? "fas fa-bookmark" : "far fa-bookmark";
+        const disabledClass = LRR.isUserLogged() ? "" : " disabled";
+        bookmarkIcon = `<i id="${id}" class="${bookmarkClass} thumbnail-bookmark-icon${disabledClass}" title="Toggle Bookmark" ${!LRR.isUserLogged() ? 'style="opacity: 0.5; cursor: not-allowed;"' : ''}></i>`;
```

Comment:
I'd put the style as a separate const as well since you did it for disabledClass.